File: Coding/03_Beginner/Full_search/JavaScript/16.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/).map(Number);
const [N, M, K] = input;
let A = input.slice(3, 3 + N);
let B = input.slice(3 + N);

let cnt = 0;
for (let i = 0; i < N; i++) {
  for (let j = 0; j < M; j++) {
    if (A[i] + B[j] == K) cnt++;
  }
}

console.log(cnt);

File: Coding/03_Beginner/Full_search/C++/16.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
    int N, M, K;
    cin >> N >> M >> K;
    vector<int> A(N), B(M);

    for (int i = 0; i < N; i++) cin >> A[i];
    for (int j = 0; j < M; j++) cin >> B[j];

    int cnt = 0;
    for (int i = 0; i < N; i++){
        for (int j = 0; j < M; j++){
            if (A[i] + B[j] == K) cnt++;
        }
    }

    cout << cnt << endl;
    return 0;
}